src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,64 @@
+import classnames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  searchResults: Person[] | null;
+  onSelect: (person: Person) => void;
+  setIsDropdownActive: (isActive: boolean) => void;
+  activePersonSlug: string | null;
+
+};
+
+export const DropdownMenu: React.FC<Props> = (
+  {
+    searchResults,
+    onSelect,
+    setIsDropdownActive,
+    activePersonSlug,
+  },
+) => {
+  if (!searchResults) {
```

Comment:
Change the condition to check for searchResults.length === 0 instead of !searchResults, as searchResults is of type Person[] | null.